Xamarin.Essentials/Calendars/Calendars.android.cs

```diff
@@ -589,7 +590,8 @@ static RecurrenceRule GetRecurranceRuleForEvent(string rule)
                         {
                             recurranceRule.Frequency = RecurrenceFrequency.YearlyOnDay;
                         }
-                        recurranceRule.WeekOfMonth = (IterationOffset)Convert.ToInt32(iterationOffset.Value);
+                        int.TryParse(iterationOffset.Value.Split(',').FirstOrDefault(), out var result);
```

Comment:
Is there a risk that value could be null?

Reply:
Adjust the condition to use the regex instead of isdigit

Reply:
Alter accordingly